src/apps/relay/dtls_listener.c

```diff
@@ -633,6 +634,11 @@ static void udp_server_input_handler(evutil_socket_t fd, short what, void *arg)
   int conn_reset = is_connreset();
   int to_block = would_block();
 
+#if defined(WINDOWS)
```

Comment:
Why add this code?

Reply:
This code is to restore the socket to blocking mode, to mimic the behavior on non-Windows builds.